scripts/gen_vimdoc.py

```diff
@@ -53,11 +53,19 @@
 from xml.dom import minidom
 
 MIN_PYTHON_VERSION = (3, 6)
+MIN_DOXYGEN_VERSION = (1, 9, 0)
 
 if sys.version_info < MIN_PYTHON_VERSION:
     print("requires Python {}.{}+".format(*MIN_PYTHON_VERSION))
     sys.exit(1)
 
+doxygen_version = tuple([int(i) for i in subprocess.check_output(["doxygen", "-v"],
+                        universal_newlines=True).split('.')])
+
+if doxygen_version < MIN_DOXYGEN_VERSION:
+    print("requires Doxygen {}.{}.{}+".format(*MIN_DOXYGEN_VERSION))
```

Comment:
Also print the users' doxygen version?

Reply:
https://github.com/neovim/neovim/pull/18356